pkg/registry/apis/provisioning/resources/dualwriter.go

```diff
@@ -26,6 +26,14 @@ type DualReadWriter struct {
 	access  authlib.AccessChecker
 }
 
+type DualWriteOptions struct {
```

Comment:
Adding the 5th parameter that gets passed to all functions seemed like it was worth making a struct